CHANGELOG.md

```diff
@@ -74,6 +74,21 @@ _**Meltano 2.0** is a major update with multiple new features and some breaking
 - Improved [telemetry](https://handbook.meltano.com/data-team/telemetry#philosophy-of-telemetry) coverage across the product.
 
 
+## 1.105.0 - (2022-05-19)
+---
+
+### New
+
+### Changes
+- [#3174](https://gitlab.com/meltano/meltano/-/issues/3174) - We now recommend pipx over manually created virtual environments.  
+- [#3450](https://gitlab.com/meltano/meltano/-/issues/3450) - Windows instllation instructions added
+- [#3367](https://gitlab.com/meltano/meltano/-/issues/3367) - Windows now covered in CI for some basic cases
```

Comment:
Should this content be moved up to the unreleased section?

Reply:
Done!